logistics_planning_invoicing/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    'name': 'Logistics Planning Invoicing',
+    'summary': '''
+        Adds link with Invoicing
```

Comment:
Enhance this description:
```suggestion
        Links Logistics Planning with Invoicing
```